flake-module.nix

```diff
@@ -120,6 +120,15 @@ ihpFlake:
                     includeDevTools = false;
                     optimized = false;
                 };
+
+                migrate =
+                    let
+                        projectSrc = pkgs.nix-gitignore.gitignoreSource [] cfg.projectPath;
```

Comment:
This is supposed to get the root path of the IHP project, yes? You could get that via a `self` argument in the arguments in line 11, I believe

Reply:
How exactly would the code look like here then?

Reply:
I think this line is not needed anymore since we don't cd into the `projectSrc`

Reply:
```nix
migrate =  pkgs.writeScriptBin "migrate" ''
    ${ghcCompiler.ihp}/bin/migrate
  '';
`